corehq/apps/auditcare/tests/test_middleware.py

```diff
@@ -126,21 +153,33 @@ def test_make_admin_view_class():
     eq(func.__module__, "django.contrib.admin")
 
 
+class FakeAuditDoc(Config):
+
+    def save(self):
+        if "save_count" in self:
+            self.save_count += 1
+        else:
+            self.save_count = 1
+
+
 ARGS = ()  # positional view args are not audited, therefore are empty
 KWARGS = {"non": "empty", "and": "audited", "view": "kwargs"}
-EXPECTED_AUDIT = Config(user="username", view_kwargs=KWARGS)
+EXPECTED_AUDIT = FakeAuditDoc(user="username", view_kwargs=KWARGS)
 Settings = Config(
     AUDIT_MODULES=default_settings.AUDIT_MODULES,
     AUDIT_VIEWS=default_settings.AUDIT_VIEWS,
 )
 
 
 @contextmanager
-def configured_middleware(settings=Settings):
+def configured_middleware(settings=Settings, get_response=None):
```

Comment:
This is the standard name used all over the [Django middleware docs](https://docs.djangoproject.com/en/3.1/topics/http/middleware/#writing-your-own-middleware) and code. It may have even originated in WSGI (the Python web server protocol).

Reply:
Good to know...I'll retract that point, then!